server/weaverbird/backends/pandas_executor/steps/formula.py

```diff
@@ -0,0 +1,15 @@
+from pandas import DataFrame
+
+from weaverbird.backends.pandas_executor.types import DomainRetriever, PipelineExecutor
+from weaverbird.formula import clean_formula, eval_formula
```

Comment:
clean_formula may stay out of a specific backend, but IMO it seems appropriate to move eval_formula inside the pandas_backend. Maybe in the weaverbird.backends.pandas_executor.steps.utils module?